### PR TITLE
Creating a Generate and Clean db script

### PR DESCRIPTION
**Pull request by agutierrez-eb**
In this PR a Create and Clean script is created. Also, the event factory is modified to match the model.

### CODE REVIEW

**Review by fgomez-eb — DISMISSED** [review]
LGTM

File: invoicing_app/management/commands/create_dummy_data.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        for index in range(quantity):
+            order = OrderFactory.create(
+                status=100,
+                pp_date=date,
+                changed=date,
+                event=event_list[index],
+                mg_fee=5.1,
+                gross=1.1,
+                eb_tax=1.1,
+            )
```
Comment (juanvallejo-eb) [inline invoicing_app/management/commands/create_dummy_data.py]: You can avoid using loops by using the Factory `create_batch` method and using size=quantity
see https://factoryboy.readthedocs.io/en/latest/reference.html#factory.create_batch

File: invoicing_app/management/commands/create_dummy_data.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+    def insert_series_event(self, user_1):
+        event_parent_1 = EventFactory.build(
+            event_name='event_parent_1',
+            user=user_1,
+            currency='ARS'
+        )
+        event_parent_1.save()
```
Comment (juanvallejo-eb) [inline invoicing_app/management/commands/create_dummy_data.py]: I think you could just use:
```
event_parent_1 = EventFactory.create(
            event_name='event_parent_1',
            user=user_1,
            currency='ARS'
)
```

File: invoicing_app/management/commands/create_dummy_data.py (hunk 1 above)
Comment (agutierrez-eb) [reply invoicing_app/management/commands/create_dummy_data.py]: We tried to use create_batch but had some problems with the object's id so, for do it faster, decided apply loops.

File: invoicing_app/management/commands/create_dummy_data.py (hunk 2 above)
Comment (agutierrez-eb) [reply invoicing_app/management/commands/create_dummy_data.py]: I'll do it!

File: invoicing_app/management/commands/create_dummy_data.py (hunk 1 above)
Comment (juanvallejo-eb) [reply invoicing_app/management/commands/create_dummy_data.py]: I see, in that case you could merge those loops into only one.

Comment (agutierrez-eb) [reply invoicing_app/management/commands/create_dummy_data.py]: You're right! I'll fix it!

**Review by juanvallejo-eb — APPROVED** [review]
looks good 👌

**Pull request merged**